specification/ai/data-plane/Azure.AI.Agents/preview/2024-07-01-preview/azure-ai-projects.json

```diff
@@ -0,0 +1,8196 @@
+{
```

Comment:
I think this file should be deleted (projects swagger under Agents filer?)

Reply:
Removed.